### PR TITLE
Update to v2.1.0

### PR DESCRIPTION
**Pull request by marcoesters**
menuinst 2.1.0

**Destination channel:** defaults

### Links

- [PKG-4849](https://anaconda.atlassian.net/browse/PKG-4849) 
- [Upstream repository](https://github.com/conda/menuinst/)
- [Upstream changelog/diff](https://github.com/conda/menuinst/blob/main/CHANGELOG.md#210-2024-05-21)

### Explanation of changes:

- Update version to 2.1.0

[PKG-4849]: https://anaconda.atlassian.net/browse/PKG-4849?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,10 +7,10 @@ package:
 source:
   fn: menuinst-{{ version }}.tar.gz
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:8]:
```suggestion
```
Already obsolete

**Comment by skupr-anaconda** [comment]
Should there be `pytest` command in `test/commands`?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -46,6 +45,9 @@ requirements:
 test:
   requires:
     - pip
+    - pytest
+  files:
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:49]:
```suggestion
  source_files:
```

**Comment by marcoesters** [comment]
> Should there be `pytest` command in `test/commands`?

Looks like they are running now. I had to skip a few tests because `defaults` is missing packages. The elevation tests do not work on Prefect.

**Review by SC426 — APPROVED** [review]

**Review by cbouss — APPROVED** [review]

**Pull request merged**